scripts/us_bls/jolts/BLSJolts_StatisticalVariables.mcf

```diff
@@ -1,2968 +1,3304 @@
-
-Node: dcid:NumJobOpening_NAICS_JOLTS_000000_Adjusted
+Node: dcid:dc/ek7gq6ylyjwn8
 typeOf: dcs:StatisticalVariable
 populationType: schema:JobPosting
 statType: dcs:measuredValue
 measuredProperty: dcs:count
-measurementQualifier: dcs:BLSSeasonallyAdjusted
+measurementQualifier: dcs:Adjusted
```

Comment:
Is this change expected?

BLSSeasonallyAdjusted is defined in schema while Adjusted is not.

Is the statvar definition is changed, pls also update schema.